redash/schedule.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+from datetime import datetime, timedelta
+from functools import partial
+from random import randint
+
+from rq_scheduler import Scheduler
+
+from redash import settings, redis_connection
+from redash.tasks import (sync_user_details, refresh_queries,
+                          empty_schedules, refresh_schemas,
+                          cleanup_query_results,
+                          version_check, send_aggregated_errors)
+
+rq_scheduler = Scheduler(connection=redis_connection,
+                         queue_name="periodic",
+                         interval=5)
+
+
+def schedule(func, **kwargs):
+    previously_scheduled_jobs = filter(lambda job: job.func == func, rq_scheduler.get_jobs())
```

Comment:
@jezdez is it wise to do this?

Reply:
Hmm, not sure if you can compare the job functions, but they should be singletons, right?

Reply:
I think unbound methods are equals, and I've debugged this and watched previously scheduled jobs get cancelled successfully. I'm more curious if you think that the concept of cancelling the previously scheduled periodic jobs is idiomatic here.